js/templates.js

```diff
@@ -1,193 +1,205 @@
-          <p class="overlay-description">
-            You can add this embed to your website by copying and pasting the HTML code below.
-          </p>
+function embedOverlay(params: Object): string {
+  // TODO(anton): This should be just one script, without iframe.
+  var script = `<script src="${window.location.origin}/embed.js"></script>`
+  var iframe = `<iframe id="charted:${params.id}" src="${params.url}" height="600px" width="100%" scrolling="yes" style="border: solid 1px #ccc"></iframe>`
+
+  return `
+    <div class="overlay-container">
+      <div class="overlay-content">
+        <h1 class="overlay-title">Embed this Charted page</h1>
+        <p class="overlay-description">
+          You can add this embed to your website by copying and pasting the HTML code below.
+        </p>
 
-          <textarea class="embed-link">${iframe}\n${script}</textarea>
+        <textarea class="embed-link">${iframe}\n${script}</textarea>
 
-          <div class="iframe-container">${iframe}</div>
-        </div>
-        <div class="overlay-close"><span class="icon icon-x"></span></div>
+        <div class="iframe-container">${iframe}</div>
       </div>
-    `
-  },
+      <div class="overlay-close"><span class="icon icon-x"></span></div>
+    </div>
+  `
+}
 
-  gridSettingsFull () {
-    return `
-      <button class="page-option-item toggle-grid" title="Show full width charts">
-        <span class="icon icon-full-screen"></span>Show full width charts
-      </button>
-    `
-  },
+function gridSettingsFull(): string {
+  return `
+    <button class="page-option-item toggle-grid" title="Show full width charts">
+      <span class="icon icon-full-screen"></span>Show full width charts
+    </button>
+  `
+}
 
-  gridSettingsSplit () {
-    return `
-      <button class="page-option-item toggle-grid" title="Show split-screen charts">
-        <span class="icon icon-split-screen"></span>Show split-screen charts
-      </button>
-    `
-  },
+function gridSettingsSplit(): string {
+  return `
+    <button class="page-option-item toggle-grid" title="Show split-screen charts">
+      <span class="icon icon-split-screen"></span>Show split-screen charts
+    </button>
+  `
+}
 
-  yAxisLabel (interval) {
-    return `
-      <div class="y-axis-label" style="top:${interval.top}">${interval.display}</div>
-    `
-  },
-
-  chart (params) {
-    var editableAttribute = ''
-    var editableButtons = ''
-
-    if (params.editable) {
-      editableAttribute = 'contenteditable="true"'
-      editableButtons = `
-        <div class="chart-options">
-          <a class="option-item toggle-type" href="#" title="Switch chart type">
-            <span class="icon icon-line"></span>
-            <span class="icon icon-column"></span>
-          </a>
+function yAxisLabel(interval: {top: string, display: string}): string {
+  return `
+    <div class="y-axis-label" style="top:${interval.top}">${interval.display}</div>
```

Comment:
This is an issue on master too, but this should add "px": `style="top:${interval.top}px"`